nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -427,6 +438,22 @@ def _impl(inputs, attr, params):
             ignores=['T'])(inputs, attr)
     return _impl
 
+def _fused_batch_norm():
+    def _impl(inputs, attr, params):
+        # Rearrange inputs from
+        # (data, gamma, beta, moving_mean, moving_variance)
+        #     to
+        # (data, gamma, beta, moving_mean, moving_varience)
```

Comment:
new_inputs is the same as inputs?

Reply:
Yes, FusedBatchNorm has same template as NNVM.
updated this unnecessary assignment.